Catappult/appcoins-wallet-android
# [APPC-2243] Evouchers: Purchase flow

**What does this PR do?**

Implements the new design of the purchase flow while keeping the current one on the iap

**Database changed?**

   No

**Where should the reviewer start?**

IabActivity.kt
AdyenPaymentPresenter.kt

**How should this be manually tested?**

On IabActivity you have a function that returns if should use bottom sheet or not, change that function to return the test case. Also change other conditions that may depend on the type of transaction since if the transaction is VOUCHER no payment methods will be returned. 

**What are the relevant tickets?**

https://aptoide.atlassian.net/browse/APPC-2243

**Code Review Checklist**
